sulong/projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/nodes/asm/syscall/linux/aarch64/LinuxAArch64Syscall.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates.
+ * Copyright (c) 2020, Arm Limited. All rights reserved.
```

Comment:
Please drop the "All rights reserved." In the BSD header this is superfluous since its already there just 2 lines below.

Reply:
Done.